dbms/src/Storages/Page/V3/CheckpointFile/Proto/manifest_file.proto

```diff
@@ -87,3 +87,14 @@ message LockFile {
     bytes name = 1;
 }
 
+message FAPTempSegmentInfo {
+    bytes segment_meta = 1;
+    bytes delta_meta = 2;
+    bytes stable_meta = 3;
+}
+
+message CheckpointIngestInfoPersisted {
+    bytes region_info = 1;
+    repeated FAPTempSegmentInfo segments = 2;
+    uint64 remote_store_id = 3;
+}
```

Comment:
IMO, it is not a proper place to put the protobuf struct for FAP. Maybe we need to define a .proto under `Storages/KVStore/MultiRaft/Disagg/`

Reply:
fixed